hack/minikube-rook-setup.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+set -x
+set -e -o pipefail
+
+## Variables
+PROFILE="${PROFILE:-hub}"
+POOL_NAME="minikube"
+
+IMAGE_DIR="${IMAGE_DIR:-"/var/lib/libvirt/images"}"
+IMAGE_NAME="osd0"
+
+ROOK_SRC="${ROOK_SRC:-"https://raw.githubusercontent.com/rook/rook/release-1.6/cluster/examples/kubernetes/ceph/"}"
+
+## Usage
+usage()
+{
+	set +x
+	echo "Usage:"
+	echo "  $0 [create|start|stop|delete]"
+	echo "  Available environment variables:"
+	echo "    minikube PROFILE ${PROFILE}"
+	echo "    minikube kvm2 image dir IMAGE_DIR ${IMAGE_DIR}"
+	echo "    rook source ROOK_SRC ${ROOK_SRC}"
+	exit 0
+}
+
+function wait_for_ssh() {
+    local tries=60
+    while ((tries > 0)); do
+        if minikube ssh "echo connected" --profile="$1" &>/dev/null; then
+            return 0
+        fi
+        tries=$((tries - 1))
+        sleep 1
+    done
+    echo ERROR: ssh did not come up >&2
+    exit 1
+}
+
+set +x
+echo "Using environment:"
+echo "  minikube PROFILE ${PROFILE}"
+echo "  minikube kvm2 image dir IMAGE_DIR ${IMAGE_DIR}"
+echo "  rook source ROOK_SRC ${ROOK_SRC}"
+set -x
+
+if [[ $1 == "delete" ]]
+then
+	minikube delete --profile="${PROFILE}"
+	virsh vol-delete --pool "${POOL_NAME}" "${IMAGE_NAME}-${PROFILE}"
+    exit 0
+fi
+
+if [[ $1 == "stop" ]]; then
+	minikube stop --profile="${PROFILE}"
+    exit 0
+fi
+
+if [[ $1 == "start" ]]; then
+	sudo virsh start "${PROFILE}"
+	wait_for_ssh "$PROFILE"
+	minikube ssh "sudo mkdir -p /mnt/vda1/rook/ && sudo ln -sf /mnt/vda1/rook/ /var/lib/rook" --profile="${PROFILE}"
+	minikube start --profile="${PROFILE}"
+    exit 0
+fi
+
+if [[ $1 != "create" ]]; then
+	usage
```

Comment:
`usage()` `exit`s with a value of `0`.  In this case, I suggest it return a non-zero value, perhaps `1`, to indicate failure to caller, or simply proceed, if `create` is default action.